test/core/test/__snapshots__/jest-expect.test.ts.snap

```diff
@@ -728,13 +900,13 @@ exports[`toSatisfy() > error message 3`] = `
 + Received
 
   {
--   "value": toSatisfy<(value) => value % 2 !== 0>,
+-   "value": toSatisfy<[Function isOdd]>,
 +   "value": 2,
   }",
   "expected": "Object {
-  "value": toSatisfy<(value) => value % 2 !== 0>,
+  "value": toSatisfy<[Function isOdd]>,
```

Comment:
It prints the function name instead of its implementation

Reply:
This is alright since this is how we normally format the function, so I guess we can say it's a better alignment.